packages/pn-pa-webapp/public/conf/env/config.dev.json

```diff
@@ -16,5 +16,6 @@
   "IS_MANUAL_SEND_ENABLED": true,
   "API_B2B_LINK": "https://developer.pagopa.it/send/api#/",
   "IS_STATISTICS_ENABLED": true,
-  "TAXONOMY_SEND_URL":"https://docs.pagopa.it/f.a.q.-per-integratori/tassonomia-send"
+  "TAXONOMY_SEND_URL": "https://docs.pagopa.it/f.a.q.-per-integratori/tassonomia-send",
+  "DOWNTIME_EXAMPLE_LINK": "https://notifichedigitali.pagopa.it/static/documents/LegalFactMalfunction.pdf"
```

Comment:
Rewrite this string. Remove the https://notifichedigitali.pagopa.it and in the configuration service compose this string with LANDING_SITE_URL